homeassistant/components/aten_pe/switch.py

```diff
@@ -0,0 +1,121 @@
+"""The ATEN PE switch component."""
+
+import logging
+import voluptuous as vol
+
+from homeassistant.components.switch import (
+    DEVICE_CLASS_OUTLET,
+    SwitchDevice,
+    PLATFORM_SCHEMA,
+)
+from homeassistant.const import CONF_HOST, CONF_PORT, CONF_USERNAME
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_COMMUNITY = "community"
+CONF_AUTHKEY = "authkey"
```

Comment:
It would be more consistent if it was more in line with the SNMP integration we already have.

```suggestion
CONF_AUTH_KEY = "auth_key"
```

Same for the `privkey`.